test/unit/specs/Getters.spec.js

```diff
@@ -5,145 +5,110 @@ import Card from '../../../src/classes/Models/Card'
 let card = new Card(55, 3, 'I', 'f')
 let card2 = new Card(34, 2, 'R', 'v')
 describe('test store.js getters', () => {
-
```

Comment:
These spaces might gives ESLint errors, I might have to change them back if they do.